velox/dwio/parquet/reader/ParquetReader.h

```diff
@@ -71,14 +71,10 @@ class ParquetReader : public dwio::common::Reader {
       const dwio::common::RowReaderOptions& options = {}) const override;
 
  private:
-  duckdb::InputStreamFileSystem* getFileSystem() {
-    static duckdb::InputStreamFileSystem fileSystem;
```

Comment:
Is the behaviour of earlier constructor based `fileSystem` variable and the one we have now via `fileSystem_ =
      std::make_unique<duckdb::InputStreamFileSystem>(std::move(stream))` identical ?
      
Unique_ptr are definitely better to use than passing around raw pointers I think.

Reply:
The behavior is the same as before. But it changed from being a singleton before to each ParquetReader instance having its own `duckdb::InputStreamFileSystem` instance.